winui/Ribbon/RibbonScreenTip.md

```diff
@@ -0,0 +1,156 @@
+                            <TextBlock Text="Like the look of a particular selection? You can apply that look to other content in the document." Width="250"
+				       TextWrapping="WrapWholeWords"/>
+                            <AppBarSeparator/>
+                            <TextBlock Text="To get started:" TextWrapping="Wrap" Width="250"/>
+                            <TextBlock Text="1. Select content with the formatting you like" TextWrapping="Wrap" Width="250"/>
+                            <TextBlock Text="2. Click Format Painter" TextWrapping="Wrap" Width="250"/>
+                            <TextBlock Text="3. Select something else to automatically apply the formatting" TextWrapping="Wrap" Width="250"/>
+                            <AppBarSeparator/>
+                            <TextBlock Text="FYI: To apply the formatting in multiple places, double-click Format Painter" 
+				TextWrapping="WrapWholeWords" Width="250"/>
+                        </StackPanel>
+                        <Border Name="PART_Seperator" Margin="4"  BorderBrush="{StaticResource ControlStrokeColorDefault}" 
+			        BorderThickness="0,0,0,1"  Grid.Row="1"  />
+                        <Grid Grid.Row="2">
+                            <Grid.ColumnDefinitions>
+                                <ColumnDefinition Width="20"/>
+                                <ColumnDefinition  />
+                            </Grid.ColumnDefinitions>
+                            <Border  HorizontalAlignment="Left"   Grid.Column="0" Margin="4,4,0,4" x:Name="HelpImage">
+                                <SymbolIcon Height="16" x:Name="helpImage" Width="16" Symbol="Help" 
+					    Foreground="{StaticResource SystemAccentColor}"/>
+                            </Border>
+                            <HyperlinkButton Content="Tell me more" FontWeight="Bold" Foreground="{StaticResource SystemAccentColor}" 
+					     NavigateUri="http://www.microsoft.com" Grid.Column="1" />
+                        </Grid>
+                    </Grid>
+                </DataTemplate>
+            </ribbon:ScreenTip.ContentTemplate>
+            <ribbon:ScreenTip.TitleTemplate>
+                <DataTemplate>
+                    <Grid>
+                        <Grid.ColumnDefinitions>
+                            <ColumnDefinition Width="Auto"/>
+                            <ColumnDefinition Width="Auto"/>
+                        </Grid.ColumnDefinitions>
+                        <TextBlock Text="Format Painter" Grid.Column="0"/>
+                        <TextBlock Text="(Ctrl+w)" Grid.Column="1" />
+                    </Grid>
+                </DataTemplate>
+            </ribbon:ScreenTip.TitleTemplate>
+        </ribbon:ScreenTip>
+    </ribbon:RibbonButton.ScreenTip>
+</ribbon:RibbonButton>
+</ribbon:RibbonGroup>
+</ribbon:RibbonTab>
+</ribbon:SfRibbon.Tabs>
+</ribbon:SfRibbon>
+
+{% endhighlight %}
+{% endtabs %}
+
+![ScreenTip in Ribbon control](Ribbon-ScreenTip-images/winui-ribbon-screentip.png)

```

Comment:
@YuvarajMasilamani, also include topic regarding title and content template customization

Reply:
Added